addons/website/static/tests/builder/website_builder/image_gallery.test.js

```diff
@@ -203,3 +205,20 @@ test("Cloning an image gallery should produce a unique ID", async () => {
     const imageCarousels = queryAll(":iframe .s_image_gallery .carousel");
     expect(imageCarousels[0].id).not.toEqual(imageCarousels[1].id);
 });
+
+test("Size option should be present when scroll down button is enabled", async () => {
+    const sizeOptionSelector = ".hb-row-sublevel-1[data-label='Size']";
+    await setupWbsiteBuilderWithImageWall(true);
+    await contains(":iframe img").click();
+    await contains("[data-label='Height'] [data-action-param='o_full_screen_height']").click();
+    await contains("[data-label='Scroll Down Button'] input").click();
+    await waitFor(sizeOptionSelector);
+    expect(`${sizeOptionSelector} .o-hb-button-group button`).toHaveCount(5);
+    await contains(`[data-class-action='fa-2x']`).click();
+    const scrollDownButtonEl = queryOne(":iframe .o_scroll_button");
+    const angleDownIconEl = scrollDownButtonEl.querySelector(".fa-angle-down");
+    const computedStyle = getComputedStyle(scrollDownButtonEl);
+    const iconFontSize = getComputedStyle(angleDownIconEl).fontSize;
+    expect(computedStyle.width).toBe(iconFontSize);
+    expect(computedStyle.height).toBe(iconFontSize);
+});
```

Comment:
The test should rather be in `tests/builder/options/spacing_option.test.js` as the spec introduces the spacing option for it, there by using `await setupWebsiteBuilderWithSnippet("any-snippet", { loadIframeBundles: true });` itself you can test the desired behavior.